bin/improver-threshold

```diff
@@ -95,6 +97,37 @@ def main():
                         "indicating a narrower fuzzy factor region / sharper "
                         "threshold. NB A fuzzy factor cannot be used with a "
                         "zero threshold or a threshold_config file.")
+    parser.add_argument("--collapse-coord", type=str,
+                        metavar="COLLAPSE-COORD", default="None",
+                        help="An optional ability to set which coordinate "
+                        "we want to collapse over. The default is set "
+                        "to None.")
+    parser.add_argument("--coord_exp_val", metavar="COORD_EXPECTED_VALUES",
+                        type=str,
+                        help="Optional string of expected coordinate points "
+                        "seperated by, e.g. '1496289600, 1496293200'.")
+    parser.add_argument("--coordinate_unit", metavar="UNIT_STRING",
+                        default="hours since 1970-01-01 00:00:00",
+                        help="Units for time coordinate. Default=hours "
+                        "since 1970-01-01 00:00:00")
+    parser.add_argument("--slope", metavar="LINEAR_SLOPE", type=float,
+                        help="The slope of the line used for choosing "
+                        "default linear weights. Only one of ynval "
+                        "and slope may be set. For realization "
+                        "collapse the default is set to 0.0")
+    parser.add_argument("--y0val", metavar="LINEAR_STARTING_POINT",
+                        type=float,
+                        help="The relative value of the weighting start "
+                        "point for choosing default linear weights. "
+                        "This must be a positive float or 0. For "
+                        "realization collapse a default of 1.0 is "
+                        "set.")
+    parser.add_argument("--weighting_mode", metavar="WEIGHTED_BLEND_MODE",
```

Comment:
The `weighted_mode` still supports the `weighted_maximum` option, as documented in the weighted-blending CLI, if someone chooses to use it. At the moment, this is undocumented but still supported.